### ISSUE TITLE
Fix CI issues for Python 3.10

### ISSUE THREAD

**Issue by om26er** [crossbario/crossbar#1965]
Python 3.10 CI is broken, it was disabled in this PR https://github.com/crossbario/crossbar/pull/1961, which make the run green. This issue is to track it so that we can re-enable Python 3.10 in CI.

**Comment by oberstet** [comment]
yeah, I tried latest pip and setuptools, but it still failed on python 3.10. no idea, and I am generally tired of this crap, hence I just moved back to 3.9
